package.tf

```diff
@@ -3,7 +3,7 @@
 data "external" "archive_prepare" {
   count = var.create && var.create_package ? 1 : 0
 
-  program     = ["python", "${path.module}/lambda.py", "prepare"]
+  program     = ["python3", "${path.module}/package.py", "prepare"]
```

Comment:
I agree with the rename into `package.py`, please update README where it is mentioned, and also update Python required version there.

Reply:
Done, a corresponding commit was fixed up.